dcos/package.py

```diff
@@ -1384,6 +1384,31 @@ def latest_version(self):
         pkg_versions.sort()
         return pkg_versions[-1]
 
+    def get_package_revision(self, package_version=None):
```

Comment:
@ConnorDoyle do you feel strongly on `package-number` instead of `package-revision`. I'll go through the code and change the names to whatever we decide to not make the code harder to read.